packages/dx-react-scheduler/docs/reference/scheduler.md

```diff
@@ -44,6 +44,21 @@ rRule? | string | Specifies the appointment recurrence rule. Follows the [iCalen
 exDate? | string | Specifies dates excluded from recurrence. Follows the [iCalendar EXDATE](https://tools.ietf.org/html/rfc5545#section-3.8.5.1) format.
 [propertyName: string] | any | Any other properties.
 
+### CellElementsMeta
+
+Field | Type | Description
+------|------|------------
+parentRect | () => ClientRect &#124; DOMRect | A function that return rect of the parent element.
+getCellRects | Array<() => ClientRect &#124; DOMRect> | An array of cell rect functions.
+
+### ScrollingStrategy
+
+Field | Type | Description
+------|------|------------
+topBoundary | number | Specify the top boundary of the scrollable layout place.
+bottomBoundary | number | Specify the bottom boundary of the scrollable layout place.
+changeVerticalScroll | (value: number) => void | A function that can change layout scrolling position.
```

Comment:
changeVerticalScroll | (value: number) => void | A function that can change layout scroll position.